user/configuration-variables.rst

```diff
@@ -0,0 +1,89 @@
+Configuration variables/context in OpenWISP2
+============================================
+
+What configuration variables are, why do we need them?
+------------------------------------------------------
+
+It's often needed for the configuration to differ a little between
+devices, we may need to use device specific values
+(like MAC address, or UUID) in it, in such situation it's not to possible
+to simply use static templates. 
+The configuration variables feature allows us to include in our config
+some default/automatically assigned variables
+and define our own ones in controller admin.
+
+The set of all configuration variables is known as *context*.
+
+Default context
+---------------
+
+Following variables are added automatically
+to the context of each configuration:
```

Comment:
replace configuration with device configuration.

Also mention the default context in templates and VPN servers, there should be something (have a look at the code of django-netjsonconfig).